js/foundation.slider.js

```diff
@@ -106,7 +106,11 @@ class Slider {
    * @fires Slider#changed
    */
   _setHandlePos($hndl, location, noInvert, cb) {
-  //might need to alter that slightly for bars that will have odd number selections.
+    // don't move if the slider has been disabled since its initialization
+    if (this.$element.hasClass(this.options.disabledClass)) {
```

Comment:
```js
var disabledAttr = this.$element.data('disabled');
var disabledClass = this.$element.hasClass(this.options.disabledClass);

if ((disabledClass || disabledAttr) && !(disabledAttr && disabledAttr == "false")) {
```

So :
- `disabled` class or attribute (not `false`) -> **Disabled**
- `disabled` option in initialization -> **Disabled**
- `disabled` option in initialization + `disabled` class or attribute (not `false`) -> **Disabled**
- `disabled` option in initialization + `disabled` class + `disabled` attribute equal to `false` -> **Enabled**